README.md

```diff
@@ -1,118 +1,106 @@
+![](docs/images/freestyle-example-scm-3.png)
 
-### Pipeline Jobs
+## Pipeline Jobs
 
-Pipeline jobs can use step `withFolderProperties` to access them either inside or outside a node step:
+Pipeline jobs can use step `withFolderProperties` to access them :
 
-    withFolderProperties{
-        echo("Foo: ${env.FOO}")
-    }
+**Using folder properties in a pipeline job**
 
-Jenkins deployments using some of the older versions of the
-[Structs Plugin](https://wiki.jenkins.io/display/JENKINS/Structs+plugin) will need to do this using the `wrap` meta-step.
-In such scenarios you should note that the wrapped syntax must run inside a node step.
+``` groovy
+withFolderProperties{
+    echo("Foo: ${env.FOO}")
+}
+```
 
-    node {
-        wrap([$class: 'ParentFolderBuildWrapper']) {
-            echo("Foo: ${env.FOO}")
-        }
-    }
+Jenkins deployments using some of the older versions of the [Structs
+Plugin](https://plugins.jenkins.io/structs/) will need
+to do this using the `wrap` meta-step :
 
-### Declarative Pipeline Jobs
+**Using folder properties in older pipeline jobs**
 
-Declarative pipeline jobs can also use the `options` directive to leverage folder properties as follows:
+``` groovy
+wrap([$class: 'ParentFolderBuildWrapper']) {
+    echo("Foo: ${env.FOO}")
+}
+```
 
-    pipeline {
-        agent any
-        options {
-            withFolderProperties()
-        }
-        stages {
-            stage('Test') {
-                steps {
-                    echo("Foo: ${env.FOO}")
-                }
-            }
-        }
-    }
+## Job DSL
 
-### Job DSL
+In Job DSL scripts you can define folder properties like so :
 
-If you use [Job DSL](https://plugins.jenkins.io/job-dsl/) you can create folders with predefined properties like this:
+**Job DSL example**
 
-    folder(folderName) {
-        properties {
-            folderProperties {
-                properties {
-                    stringProperty {
-                        key('FOO')
-                        value('bar')
-                    }
+``` groovy
+folder('my folder') {
+    properties {
+        folderProperties {
+            properties {
+                stringProperty {
+                    key('FOO')
+                    value('foo1')
                 }
             }
         }
     }
+}
+```
 
-## Authors & Contributors:
+## Authors & Contributors 
 
 * [Miguelángel Fernández Mendoza](https://github.com/mig82).
 * [GongYi](https://github.com/topikachu).
 * [Stefan Hirche](https://github.com/StefanHirche)
+* [Deepak Gupta](https://github.com/Mr-DG-Wick)
 
 ## References
 
 * Site: https://mig82.github.io/folder-properties-plugin
 * Dependencies: https://mig82.github.io/folder-properties-plugin/doc/dependencies.html
-* Javadoc: https://mig82.github.io/folder-properties-plugin/apidocs
+* Javadoc: https://mig82.github.io/folder-properties-plugin/apidocs
```

Comment:
@mig82 any reason this could not use the standard Jenkins javadoc location instead of using your GitHub pages version?
```suggestion
* Javadoc: https://javadoc.jenkins-ci.org/plugin/folder-properties/
```